src/components/TeamSection/TeamInformation.ts

```diff
@@ -31,11 +31,21 @@ export const executiveTeam: TeamMember[] = teamData.executiveTeam.map((exec, ind
     image: importImage(exec.imagePath)
   }));
 
-export const projectManagers: TeamMember[] = teamData.projectManagers.map((pm, index) => ({
-  id: index,
-  ...pm,
-  image: importImage(pm.imagePath)
-}));
+export const projectManagers: TeamMember[] = teamData.projectManagers.length > 0 ? 
```

Comment:
This is a bit shorter and lets `projectManagers` be undefined without breaking
```suggestion
export const projectManagers: TeamMember[] = teamData.projectManagers?.length ? 
```